src/plugins/dam/components/uploader/FileList.jsx

```diff
@@ -32,22 +33,11 @@ class FileList extends React.Component {
     super(props);
 
     this.handleOnChange = this.handleOnChange.bind(this);
+    this.damFileList = React.createRef();
   }
 
   componentDidUpdate() {
-    // Raven is making `componentDidUpdate` fire on each heart beat thus
-    // the uploader modal jumps when users with a small screen (like a 13" mbp)
-    // scrolls down. This will check if the user's browser is about a 13" screen
-    // or less and prevent the scrolling into view from happening.
-    if (document.documentElement.clientHeight > 792) {
-      this.ensureActiveItemVisible();
-    }
-  }
-
-  ensureActiveItemVisible() {
-    if (this.activeFileItem) {
-      scrollIntoViewIfNeeded(this.activeFileItem, { duration: 500 });
-    }
+   this.ensureActiveItemVisible();
```

Comment:
@digitalartcrew this is where the throttle should go, ref some other examples in teh codebase like https://github.com/triniti/cms-js/blob/656d22fc77c88d6f3269e6403048b8856847c433/src/plugins/iam/services/Authenticator.js#L275